nilearn/_utils/compat.py

```diff
@@ -58,3 +62,19 @@ def md5_hash(string):
         m = hashlib.md5()
         m.update(string)
         return m.hexdigest()
+
+
+if LooseVersion(nibabel.__version__) >= LooseVersion('2.0.0'):
+    def get_affine(img):
+        return img.affine
+else:
+    def get_affine(img):
+        return img.get_affine()
+
+
+if LooseVersion(nibabel.__version__) >= LooseVersion('2.0.0'):
+    def get_header(img):
+        return img.header
+else:
+    def get_header(img):
+        return img.get_header()
```

Comment:
Complete nitpick (sorry, I miss it earlier): could you please join the 2 if/else blocks above into one, that defines the 2 functions at once.